addons/mass_mailing/wizard/mailing_contact_import.py

```diff
@@ -0,0 +1,121 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, tools, _
+from odoo.tools.misc import clean_context
+
+
+class MailingContactImport(models.TransientModel):
+    _name = 'mailing.contact.import'
+    _description = 'Mailing Contact Import'
+
+    @api.model
+    def default_get(self, fields_list):
+        vals = super(MailingContactImport, self).default_get(fields_list)
+        active_id = self._context.get('active_id')
+        if active_id and self.env.context.get('active_model') == 'mailing.mailing':
+            active_mailing = self.env['mailing.mailing'].browse(active_id)
+            if active_mailing.contact_list_ids:
+                vals['list_ids'] = active_mailing.contact_list_ids[0]
+        return vals
+
+    list_ids = fields.Many2many('mailing.list', string='Lists')
```

Comment:
Note that we could also have a "origin_mailing_id" field and make list_ids a computed editable field, that would take the mailing mailing lists (and user can then update it). That way we only have to give ``default_origin_mailing_id`` and hopla, default_get is removed.